tables/tests/resources/test_worksheet.csv

```diff
@@ -0,0 +1,103 @@
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,TESTID4S,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,Column1,Column2,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,TESTID4E,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,TESTID5S,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,DropdownColumn,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,< Select >,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,Select from dropdown,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,< Select >,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,TESTID5E,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,TESTID6S,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,Whitespace,,,,,,,,,,,,,,,,,,,,,,,,,
+,,, leadingspace,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,trailingspace ,,,,,,,,,,,,,,,,,,,,,,,,,
+,,, ,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,TESTID6E,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,TESTID7S,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,ColumnA,ColumnB,,,,,,,,,,,,,,,,,,,,,,,
+,TESTID7S,,,5,01/01/2001,,,,,,,,,,,,,,,,,,,,,,,
+,ColumnA,ColumnB,,6,02/01/2001,,,,,,,,,,,,,,,,,,,,,,,
+,1,10/10/2010,,7,03/01/2001,,,,,,,,,,,,,,,,,,,,,,,
+,2,11/10/2010,,,,,,,,,,,,,,,,,,,,,,,,,,
+,3,12/10/2010,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,TESTID7E,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,TESTID7E,,,,,,,,,,,,,,,,,,,,,,,
+,TESTID7S,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,ColumnA,ColumnB,,,,,,,,,,,,,,,,,,,,,,,,,,
+,0,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,11/11/2011,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,TESTID7E,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,TESTID8S,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,ColumnInSchema1,DroppedColumn,ColumnInSchema2,,,,,,,,,,,,,,,,,,,,,,,,,
+,A,D,G,,,,,,,,,,,,,,,,,,,,,,,,,
+,B,E,H,,,,,,,,,,,,,,,,,,,,,,,,,
+,C,F,I,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,TESTID8E,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,TESTID10S,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,TopHeader,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,BottomHeader1,BottomHeader2,BottomHeader3,,,,,,,,,,,,,,,,,,,,,,,,,
+,A,B,C,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,TESTID10E,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,TESTID11S,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,TopHeader,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,MiddleHeader1,MiddleHeader2,,,,,,,,,,,,,,,,,,,,,,,,,,
+,BottomHeader1,BottomHeader2,BottomHeader3,,,,,,,,,,,,,,,,,,,,,,,,,
+,A,B,C,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,TESTID11E,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,TESTID12S,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,Column1,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,A,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,TESTID13S,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,Column1,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,A,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,TESTID13E,,,,,,,,,,,,,,,,,,,,,,,,,,
```

Comment:
Apologies if I've missed something similar, but a good test case might be:
```
col1,col2,col3
useful_data,useful_data,useful_data
guidance notes, guidance notes, guidance notes
useful_data,useful_data,useful_data
```
This is a pattern we might encounter?

Reply:
Are we expecting all tables to be vertical like a DataFrame (with columns names on top of all data) from now on? Is there not a circumstance where the columns will appear horizontally in the original sheet, and if so how do we account for this?

Reply:
To your first comment that is definitely something we may encounter but it's actually already tested here (specifically line 249). Agree it isn't that clear from the test that it can be used in the "guidance notes" context that we see often in the actual spreadsheets as the parameter is more general than that

https://github.com/communitiesuk/funding-service-design-post-award-data-store/blob/dd080490bc4f471db8e00c195adbc1b67863aa72/tables/tests/test_schema_extract.py#L242-L264


And to your second one, yeah I've been thinking about that - we'll definitely come across those style of tables. I'm planning to implement like a `transposed` flag or something, that will just flip the extracted table during processing and make the first column the header row. I think it should be ok. It will need to be implemented for Pathfinders, but I may not do it as part of this PR.